src/driver/test/Driver_test.cpp

```diff
@@ -326,4 +328,105 @@ TEST_CASE("Various Actor Behaviors") {
                  Fails::state.reachedPhases() == std::multiset<genny::PhaseNumber>{0}));
         REQUIRE(hasMetrics(opts));
     }
+
+    SECTION("Load External Config Default Parameter") {
+        auto [code, opts] = outcome(R"(
+        SchemaVersion: 2018-07-01
+        Actors:
+          - Type: Fails
+            Threads: 1
+            Phases:
+            - ExternalPhaseConfig:
+                Path: "src/phase_configs/self_tests/Good.yml"
+
+        )");
+        REQUIRE(code == DefaultDriver::OutcomeCode::kSuccess);
+        REQUIRE(Fails::state.reachedPhases() == std::multiset<genny::PhaseNumber>{0});
+        REQUIRE(hasMetrics(opts));
+    }
+
+    SECTION("Load External Config Default Parameter") {
+        auto [code, opts] = outcome(R"(
+        SchemaVersion: 2018-07-01
+        Actors:
+          - Type: Fails
+            Threads: 1
+            Phases:
+            - ExternalPhaseConfig:
+                Path: "src/phase_configs/self_tests/GoodWithKey.yml"
+                Key: ForSelfTest
+        )");
+        REQUIRE(code == DefaultDriver::OutcomeCode::kSuccess);
+        REQUIRE(Fails::state.reachedPhases() == std::multiset<genny::PhaseNumber>{0});
+        REQUIRE(hasMetrics(opts));
+    }
+
+    SECTION("Load External Config Override Parameter") {
+        auto [code, opts] = outcome(R"(
+        SchemaVersion: 2018-07-01
+        Actors:
+          - Type: Fails
+            Threads: 1
+            Phases:
+            - ExternalPhaseConfig:
+                Path: "src/phase_configs/self_tests/Good.yml"
+                Parameters:
+                  Repeat: 2
```

Comment:
I thought we didn't want `Repeat` and ilk to be external? I was imagining the syntax looking like

```yaml
- Type: Fails
  Threads: 1
  Phases:
  - ExternalPhaseConfig:
      Path: self_tests/Good.yml
    Repeat: 10
```

Rationale being that the external Phase yaml can't really both specify `Repeat` and `Duration` as parameters.

Reply:
Oh yeah. Thanks for catching this!

I'll add a test tomorrow.

Reply:
I read over Henrik's sample workload again and he wanted to reuse Duration. So I'm keeping support for `Duration` and `Repeat` in because it's also easier code-wise. The solution is to allow merging of keys specified in external yamls and workload yamls. Conflicts can be caught by `--evaluate`.

The alternative solution which I didn't do is to hard code in checks for top-level keys the external YAML can't contain.